Update requirements and docs migration (#11170)
open-metadata/OpenMetadata: openmetadata-ui/src/main/resources/ui/public/locales/en-US/Dashboard/DomoDashboard.md

In this section, we provide guides and references to use the DomoDashboard connector.

# Requirements

> For metadata ingestion, kindly make sure add alteast `dashboard` scopes to the clientId provided. Question related to scopes, click [here](https://developer.domo.com/docs/authentication/quickstart-5).

Reviewer: ```suggestion
For metadata ingestion, kindly make sure to add at least `dashboard` scopes to the clientId provided. For questions related to scopes, click [here](https://developer.domo.com/docs/authentication/quickstart-5).
```

You can find further information on the Domo Dashboard connector in the [docs](https://docs.open-metadata.org/connectors/dashboard/domo-dashboard).

## Connection Details

$$section

> ### Client Id $(id="clientId")

Reviewer: ```suggestion
### Client ID $(id="clientId")
```

> Client ID for DOMO. Further information on Client ID can be found [here](https://docs.open-metadata.org/connectors/database/domo-database/troubleshoot#how-to-find-clientid)

Reviewer: ```suggestion
Client ID for DOMO. Further information on Client ID can be found [here](https://docs.open-metadata.org/connectors/database/domo-database/troubleshoot#how-to-find-clientid).
```

$$section
### Access Token $(id="accessToken")

> Access token to connect to DOMO. Further information on Client ID can be found [here](https://docs.open-metadata.org/connectors/database/domo-database/troubleshoot#where-to-find-accesstoken)

Reviewer: ```suggestion
Access token to connect to DOMO. Further information on Client ID can be found [here](https://docs.open-metadata.org/connectors/database/domo-database/troubleshoot#where-to-find-accesstoken).
```

$$

$$section

> ### Api Host $(id="apiHost")

Reviewer: ```suggestion
### API Host $(id="apiHost")
```

> API Host to connect to DOMO instance. Default value here is `api.domo.com`

Reviewer: ```suggestion
API Host to connect to DOMO instance. Default value here is `api.domo.com`.
```

$$

$$section
### Sandbox Domain $(id="sandboxDomain")

Connect to Sandbox Domain. For example `https://<your>.domo.com`

> <!-- sandboxDomain to be updated -->

Reviewer: ```suggestion
```
